scripts/jenkins/ardana/README.md

```diff
@@ -60,7 +60,16 @@ inside the stage steps, which would show the stage as being always executed, reg
 
 ## Permitted Jenkinsfile contents
 
-The contents of a Jenkinsfile are restricted to the following:
+Jenkinsfiles are basically Groovy scripts and while there is no way around using Jenkinsfiles
```

Comment:
Jenkinsfiles are DSLs with  Groovy-like syntax https://jenkins.io/doc/book/pipeline/getting-started/

Reply:
> Jenkinsfiles are DSLs with Groovy-like syntax https://jenkins.io/doc/book/pipeline/getting-started/
@aojea yes, that is true. I corrected the text and rephrased some of the ideas to hopefully make it clearer